File: server/Entities/MultiAssist.cs
using System;
using System.Collections.Generic;

namespace WebApi.Entities {
	public class MultiAssists {

        public MultiAssists(){
            Beneficiaries = new HashSet<Beneficiaries>();
            MultiAssistsVehicle = new HashSet<MultiAssistsVehicle>();
        }

		public int Id { get; set; }
		public int ClientProductId { get; set; }
		public int CoverId { get; set; }
		public DateTime? EffectiveDate { get; set; }
		public DateTime? EligibleWaitingPeriodDate { get; set; }
        public DateTime? EndDate { get; set; }
        public DateTime? SentDate { get; set; }
        public double? Cost { get; set; }
        public string? Ref1 { get; set; }
        public string? Ref2 { get; set; }
        public string? Ref3 { get; set; }
		public string StatusId { get; set; }
        public string? AccountType { get; set; }
        public string? BankName { get; set; }
        public string? AccountHolderName { get; set; }
        public string? RoutingNum { get; set; }
        public string? AccountNum { get; set; }
        public string? ExpDate { get; set; }
        public int? DebDay { get; set; }

        public string? DebRecurringType { get; set; }

		public DateTime? CreatedAt { get; set; }
		public DateTime? UpdatedAt { get; set; }
		public DateTime? DeletedAt { get; set; }
        public virtual ClientProduct ClientProduct { get; set; }
        public virtual Covers Cover { get; set; }

        public virtual ICollection<Beneficiaries> Beneficiaries { get; set; }

        public virtual ICollection<MultiAssistsVehicle> MultiAssistsVehicle { get; set; }


	}
}

File: client-admin/src/app/models/MultiAssist.ts
import { faMailBulk } from '@fortawesome/free-solid-svg-icons';

export class MultiAssist {
  id: number;
  coverId: number;
  effectiveDate: Date;
  eligibleWaitingPeriodDate: Date;
  endDate: Date;
  cost: number;
  statusId: number;
  accountType: string;
  bankName: string;
  accountHolderName: string;
  routingNum: string;
  accountNum: string;
  expDate: Date;
  debDay: number;
  debRecurringType: string;
  beneficiaries: Array<Beneficiaries>;
  constructor(
    id: number,
    coverId: number,
    effectiveDate: Date,
    eligibleWaitingPeriodDate: Date,
    endDate: Date,
    cost: number,
    statusId: number,
    accountType: string,
    bankName: string,
    accountHolderName: string,
    routingNum: string,
    accountNum: string,
    expDate: Date,
    debDay: number,
    debRecurringType: string,
    beneficiaries: Array<Beneficiaries>
  ) {
    this.id = id;
    this.coverId = coverId;
    this.effectiveDate = effectiveDate;
    this.eligibleWaitingPeriodDate = eligibleWaitingPeriodDate;
    this.endDate = endDate;
    this.cost = cost;
    this.statusId = statusId;
    this.accountType = accountType;
    this.bankName = bankName;
    this.accountHolderName = accountHolderName;
    this.routingNum = routingNum;
    this.accountNum = accountNum;
    this.expDate = expDate;
    this.debDay = debDay;
    this.debRecurringType = debRecurringType;
    this.beneficiaries = beneficiaries;
  }
}

export class Beneficiaries {
  id: number;
  name: string;
  birthDate: Date;
  relationship: string;
  percent: string;
  ssn: string;
  multiAssistId: number;
  /**
   *
   */
  constructor(
    id: number,
    name: string,
    birthDate: Date,
    relationship: string,
    percent: string,
    ssn: string,
    multiAssistId: number
  ) {
    this.id = id;
    this.name = name;
    this.birthDate = birthDate;
    this.relationship = relationship;
    this.percent = percent;
    this.ssn = ssn;
    this.multiAssistId = multiAssistId;
  }
}

export class MultiAssistsVehicle {
  id: number;
  multiAssistId: number;
  make: string;
  model: string;
  year: number;
  vin: string;
  /**
   *
   */
  constructor(
    id: number,
    multiAssistId: number,
    make: string,
    model: string,
    year: number,
    vin: string
  ) {
    this.id = id;
    this.multiAssistId = multiAssistId;
    this.make = make;
    this.model = model;
    this.year = year;
    this.vin = vin;
  }
}
